homeassistant/components/velbus/climate.py

```diff
@@ -78,3 +92,11 @@ async def async_set_preset_mode(self, preset_mode: str) -> None:
         """Set the new preset mode."""
         await self._channel.set_preset(PRESET_MODES[preset_mode])
         self.async_write_ha_state()
+
+    @api_call
+    async def async_set_hvac_mode(self, **kwargs: str) -> None:
+        """Set the new hvac mode."""
+        if (mode := kwargs.get("hvac_mode")) is None:
+            return
+        await self._channel.set_mode(mode)
```

Comment:
What's the type of `mode` here?

Reply:
heating or cooling.. 

The Velbus bus system reports back the actual mode (heating / cooling) in the bus protocol.. we take that actual mode and set it in HA (also to accommodate switches done externally to HA on the mode)  and if switched in HA - we send the command on the bus to switch the actual device to that mode.

Reply:
I mean, is it `cool`, `heat`, `True` or `False`

Reply:
ah, indeed "cool" or "heat"